### DISCUSSION TITLE
Are Rancher Desktop logs truncated based on size?

### DISCUSSION THREAD

**Discussion by reddog335** [rancher-sandbox/rancher-desktop#2802] [General]
Are the Rancher Desktop logs in $HOME\AppData\Local\rancher-desktop\logs truncated based on size?  If not, how long are they retained.

**Comment by adamkpickering** [comment]
I believe that the existing logs are deleted at the beginning of every run. So each log file contains content from just one run. I'm not aware of anything that would truncate logs during a long term run.
